ui/app/controllers/pipeline.js

```diff
@@ -25,6 +31,12 @@ export default class PipelineController extends Controller {
 
   @action
   async stopPipeline(pipeline) {
+    try {
+      await pipeline.startPipeline();
+    } catch (error) {
+      this._handleStartStopPipelineError(error);
+      return;
+    }
     await pipeline.stopPipeline();
```

Comment:
Would this line move into the `try / catch` block or should this rather be run on its own?